pytorch_lightning/metrics/converters.py

```diff
@@ -31,10 +32,20 @@
 try:
     from torch.distributed import ReduceOp
 except ImportError:
+
     class ReduceOp:
         SUM = None
 
-    rank_zero_warn('Unsupported `ReduceOp` for distributed computing')
+    rank_zero_warn("Unsupported `ReduceOp` for distributed computing")
+
+try:
+    import torch_xla
+    import torch_xla.core.xla_model as xm
+    import torch_xla.core.functions as xf
+except ImportError:
+    XLA_AVAILABLE = False
+else:
+    XLA_AVAILABLE = True
```

Comment:
Is this used anywhere?

Reply:
Ah no. I was working on a function to also do the same on TPU, but for now, I won't include this here